packages/Chem/src/widgets/properties.ts

```diff
@@ -56,6 +85,70 @@ export function getMoleculeCharge(mol: OCL.Molecule): number {
   return moleculeCharge;
 }
 
+export async function statsWidget(molCol: DG.Column<string>): Promise<DG.Widget> {
+  const host = ui.div();
+  host.style.marginLeft = '-25px';
+  const size = Math.min(molCol.length, 1000);
+  const randomIndexes = Array.from({length: size}, () => Math.floor(Math.random() * molCol.length));
+
+  function getAverage(p: IChemProperty) {
+    const propertiesArray = new Array(size);
+    for (let idx = 0; idx < randomIndexes.length; ++idx) 
+      propertiesArray[idx] = getPropertyValue(molCol, randomIndexes[idx], p);
+    const col = DG.Column.fromList(p.type, molCol.dataFrame.columns.getUnusedName(p.name), propertiesArray);
```

Comment:
Why would we create a column here? What for, just to pass it to create a histogram? It's a bit wasteful, can it work with numbers instead? (not a major concern)

Reply:
Tried to pass number instead but it doesn't work this way. And initial method in the core takes column too